src/test/java/com/wikia/webdriver/common/logging/VelocityWrapper.java

```diff
@@ -0,0 +1,196 @@
+
+
+public class VelocityWrapper {
+
+  private static final String LAST_LOG_ROW_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/lastLogRow.vm";
+  private static final String LOG_ROW_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/logRow.vm";
+  private static final String LOG_ROW_WITH_LINK_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/logRowWithLink.vm";
+  private static final String IMAGE_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/image.vm";
+  private static final String ERROR_LOG_ROW_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/errorLogRow.vm";
+  private static final String LOG_ROW_WITH_SCREENSHOT_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/logRowWithScreenshot.vm";
+  private static final String LINK_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/link.vm";
+  private static final String FIRST_LOG_ROW_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/firstLogRow.vm";
+  private static final String BUTTON_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/button.vm";
+  private static final String HEADER_TEMPLATE_PATH = "./src/test/java/com/wikia/webdriver/common/velocitytemplates/header.vm";
+
+  public VelocityWrapper(){
+    VelocityEngine ve = new VelocityEngine();
+    ve.init();
+  }
+  static String fillLogRow(List<String> classList, String command, String description) {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate(LOG_ROW_TEMPLATE_PATH);
+    VelocityContext context = new VelocityContext();
+
+    context.put("className", "\"" + classList.stream().collect(Collectors.joining(" ")) + "\"");
+    context.put("command", command);
+    context.put("description", description);
+    StringWriter writer = new StringWriter();
+    t.merge(context, writer);
+    builder.append(writer.toString());
+    return builder.toString();
+  }
+
+  static String fillLastLogRow() {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate(LAST_LOG_ROW_TEMPLATE_PATH);
+    VelocityContext context = new VelocityContext();
+
+    StringWriter writer = new StringWriter();
+    t.merge(context, writer);
+    builder.append(writer.toString());
+    return builder.toString();
+  }
+
+  public static String fillLogRowWithLink(String link, String label) {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate(LOG_ROW_WITH_LINK_TEMPLATE_PATH);
+    VelocityContext context = new VelocityContext();
+    context.put("link", escapeHtml(link));
+    context.put("label", escapeHtml(label));
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "label" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)